bootstrap/src/test/kotlin/jp/co/soramitsu/bootstrap/EthTest.kt

```diff
@@ -30,27 +30,75 @@ class EthTest {
+                                      gasPrice = 10000000000,
+                                      gasLimit = 4500000,
+                                      confirmationPeriod = 20
+                                  )
+                              ),
+                              masterContract = MasterContractProperties(
+                                  address = "0x7d1a4fd3d286e5eb239f7f081481ab5be3517c00"
+                              ),
+                              newPeerAddress = "0x7432fc601d81362f9492ded6d4a670bcd25970d0"
+                          )
+                      )
+                  )
+              )
+              .andExpect(MockMvcResultMatchers.status().isOk)
+              .andReturn()
+          val respBody = mapper.readValue(result.response.contentAsString, MasterContractResponse::class.java)
+
+          assertNull(respBody.errorCode)
+      }
+
     @Test
     @Ignore
-    fun testDeploySmartContract() {
+    fun testDeploySmartContractMainNet() {
         val result: MvcResult = mvc
             .perform(
                 MockMvcRequestBuilders.post("/eth/deploy/D3/smartContracts").contentType(MediaType.APPLICATION_JSON).content(
                     mapper.writeValueAsString(
                         MasterContractsRequest(
                             network = EthereumNetworkProperties(
-                                ethPasswords = EthereumPasswordsImpl(credentialsPassword = "password is specific for network creds"),
-                                ethereumConfig = EthereumConfigImpl()),
-                                notaryEthereumAccounts = listOf()
+                                ethPasswords = EthereumPasswordsImpl(
+                                    credentialsPassword = "joms...",
+                                    nodeLogin = "devel...",
+                                    nodePassword = "emooy..."
+                                ),
+                                ethereumConfig = EthereumConfigImpl(
+                                    url = "https://parity1....",
+                                    credentialsPath = "D:\\soramitsu\\D3\\notary\\bootstrap\\src\\main\\resources\\eth\\main-ent-genesis.key",
+                                    gasPrice = 10000000000,
+                                    gasLimit = 4500000,
+                                    confirmationPeriod = 20
+                                )
+                            ),
+                            notaryEthereumAccounts = listOf("0x7432fc601d81362f9924ded6d4a670bcd25970d0")
                         )
                     )
                 )
             )
             .andExpect(MockMvcResultMatchers.status().isOk)
             .andReturn()
         val respBody = mapper.readValue(result.response.contentAsString, MasterContractResponse::class.java)
-
+        log.info("DeployResponse: " + result.response.contentAsString)
```

Comment:
Can be changed to `log.info("DeployResponse:  ${result.response.contentAsString}")`

Reply:
Done, whatever